### ISSUE TITLE
Build fail

### ISSUE THREAD

**Issue by guimard** [shelljs/shelljs#929]
### Node version (or tell us if you're using electron or some other framework):
10.15

### ShellJS version (the most recent version/Github branch you see the bug on):
0.8.3

### Operating system:
Linux Debian unstable

### Description of the bug:
Test fails when launching `ava test/*.js`

```
  541 passed
  5 failed

  exec › set maxBuffer (very small)
  /home/xavier/dev/debian/packages/node-shelljs/test/exec.js:137

   136:   shell.exec('echo 1234567890', { maxBuffer: 6 });
   137:   t.truthy(shell.error());                        
   138: });                                               

  Value is not truthy:

  null



  exec › set timeout option
  /home/xavier/dev/debian/packages/node-shelljs/src/exec.js:89

   88:   } else {                                              
   89:     stdout = fs.readFileSync(stdoutFile, opts.encoding);
   90:     stderr = fs.readFileSync(stderrFile, opts.encoding);

  Error thrown in test:

  ShellJSInternalError (Error) {
    code: 'ENOENT',
    errno: -2,
    path: '/tmp/shelljs_4d132e669300c1875c30',
    syscall: 'open',
    message: 'ENOENT: no such file or directory, open \'/tmp/shelljs_4d132e669300c1875c30\'',
  }

  execSync (src/exec.js:89:17)
  Object._exec (src/exec.js:205:12)
  Object.exec (src/common.js:335:23)
  Test.t [as fn] (test/exec.js:144:9)



  exec › command that fails
  /home/xavier/dev/debian/packages/node-shelljs/test/exec.js:232

   231:   shell.exec('shx cp onlyOneCpArgument.txt', { silent: true }, (code, stdout, stderr) => {
   232:     t.is(code, 1);                                                                        
   233:     t.is(stdout, '');                                                                     

  Difference:

  - 127
  + 1



  pipe › Synchronous exec
  /home/xavier/dev/debian/packages/node-shelljs/test/pipe.js:64

   63:   const result = shell.cat('test/resources/grep/file').exec('shx grep "alpha*beta"');
   64:   t.falsy(shell.error());                                                            
   65:   t.is(result.toString(), 'alphaaaaaaabeta\nalphbeta\n');                            

  Value is not falsy:

  'exec: '



  pipe › Asynchronous exec
  /home/xavier/dev/debian/packages/node-shelljs/test/pipe.js:70

   69:   shell.cat('test/resources/grep/file').exec('shx grep "alpha*beta"', (code, stdout) => {
   70:     t.is(code, 0);                                                                       
   71:     t.is(stdout, 'alphaaaaaaabeta\nalphbeta\n');                                         

  Difference:

  - 127
  + 0
```

**Comment by nfischer** [comment]
Thanks for the report. This is something we should definitely look into.

I can repro a couple failures on my debian box (I think they're related to the default `chmod` value for the system). I regularly run tests on my ubuntu box without issue.

Could you try on a variety of node versions and let me know which failures you get on each?

**Comment by nfischer** [comment]
Also, by `0.8.3`, I assume you mean the latest master branch. Correct me if I'm mistaken.

**Comment by guimard** [comment]
@nfischer: 0.8.3 = version published on https://www.npmjs.com/package/shelljs

**Comment by nfischer** [comment]
We don't ship our tests to npm. I thought this was a complaint our tests didn't work.

Please provide exact commands for how you're downloading shelljs and running tests. Or, explain the real issue if I've misunderstood.

**Comment by guimard** [comment]
* download 0.8.3 from https://github.com/shelljs/shelljs/archive/v0.8.3.tar.gz
 * `npm install`
 * `npm run test`

This gives : [logs.txt](https://github.com/shelljs/shelljs/files/2949788/logs.txt)

**Comment by nfischer** [comment]
Why are you running tests from the 0.8.3 tag? Can you run on master instead?

The logs in https://github.com/shelljs/shelljs/issues/929#issuecomment-471324526 don't match the logs from the initial report. The logs in the comment are for #915, which we fixed after v0.8.3 branched.

It sounds like there's nothing to fix?
